data/base-crafting.yaml

```diff
@@ -891,24 +891,32 @@ remedies:
     << : *ilithi_remedies
 artificing:
   Crossing:
-    npc: Enchanting
-    npc_last_name: Trainer
+    npc: Trainer
+    npc_last_name: 
```

Comment:
npc section of base-crafting.yaml needs to be below, otherwise it wont find the trainer to ask for workorders.

```
npc: Trainer
npc_last_name: Trainer
```